yoga/YGEnums.h

```diff
@@ -14,28 +14,54 @@
 YG_EXTERN_C_BEGIN
 
 #define YGFlexDirectionCount 4
+#ifndef NS_ENUM
```

Comment:
Could we make this generate something short (less duplicated) ? Maybe by creating our own YG_ENUM macro which expands to one of the two alternatives.